editor/src/components/Toc/ArticleStepList.tsx

```diff
@@ -124,14 +124,15 @@ function ArticleStepList() {
   } = useSelector((state: RootState) => state.toc);
```

Comment:
I think it would be better to add a initial value (`[]`) for `articleStepList` here.